docs/src/bpmn-workflows/message-events/message-events.md

```diff
@@ -36,6 +36,8 @@ When the activity is entered then it creates a corresponding message subscriptio
 
 A message can be referenced by one or more message events. It **must** define the name of the message (e.g. `Money collected`) and the `correlationKey` expression (e.g. `= orderId`). If the message is only referenced by message start events then the `correlationKey` is not required.
 
+Usually, the name of the message is defined as a static value (e.g. `order canceled`) but it can also be defined as [expression](/reference/expressions.html) (e.g. `= "order " + awaitingAction`). If the expression belongs to a message start event of the workflow then it is evaluated on deploying the workflow. Otherwise, it is evaluated on activating the message event. The evaluation must result in a `string`.
```

Comment:
```suggestion
Usually, the name of the message is defined as a static value (e.g. `order canceled`), but it can also be defined as [expression](/reference/expressions.html) (e.g. `= "order " + awaitingAction`). If the expression belongs to a message start event of the workflow, then it is evaluated on deploying the workflow. Otherwise, it is evaluated on activating the message event. The evaluation must result in a `string`.
```
I'm only missing some commas here.